source/configuration_manual/shared_mailboxes/cluster_setup.rst

```diff
@@ -62,6 +62,9 @@ Additionally imapc must be configured accordingly on the backends:
    configured on all backends and directors
  * :ref:`setting-imapc_host` must point to a load balancer's address that
    connects to Dovecot director/proxy
+ * :ref:`setting-imapc_features`: To be able to pass through ``SEARCH`` commands
+   to the sharing users backend :ref:`setting-imapc_features` should at least
```

Comment:
`users'` I think?

Reply:
it now has `user's` but timo suggested `users'`

Reply:
user's looks to be more correct actually. But a comma would clarify in `user's backend, imapc_features ...`